packages/hooks/src/__tests__/useWalletsList.spec.tsx

```diff
@@ -101,4 +101,46 @@ describe('useWalletsList', () => {
 
         expect(result.current.data?.map(wallet => wallet.currency)).toEqual(['AUD', 'BTC', 'ETH', 'UST', 'USD']);
     });
+
+    test('should return has_wallet equals to true if the client has at least one wallet', () => {
+        const mock = mockStore({ client: { accounts: { CRW909900: { token: '12345' } }, loginid: 'CRW909900' } });
+
+        const wrapper = ({ children }: { children: JSX.Element }) => (
+            <APIProvider>
+                <StoreProvider store={mock}>{children}</StoreProvider>
+            </APIProvider>
+        );
+
+        const { result } = renderHook(() => useWalletsList(), { wrapper });
+
+        expect(result.current.has_wallet).toEqual(true);
+    });
+
+    test("should return has_wallet equals to false if the client doesn't have any wallet", () => {
+        const mock = mockStore({ client: { accounts: { CR123456: { token: '12345' } }, loginid: 'CR123456' } });
+
+        (useFetch as jest.Mock).mockReturnValue({
```

Comment:
@heorhi-deriv Follow the mocking approach in the `useCurrencyConfig` test file, By using `as jest.Mock` we lose TS.